checkin/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+class Events(models.Model): 
+    #event_id will store spreadsheet link for unique identification of event 
+    event_id = models.CharField(max_length=500) 
+    event_name = models.CharField(max_length=100) 
+    total_attendees = models.IntegerField(default=0) 
+    check_in_count = models.IntegerField(default=0) 
+    created_date = models.DateTimeField(blank=True, null=True) 
+    updated_date = models.DateTimeField(blank=True, null=True) 
```

Comment:
It should be:
`created_at`
`updated_at`
Because it's datetime field and we can't use `created_date`